sentry-core/src/main/java/io/sentry/core/DirectoryProcessor.java

```diff
@@ -0,0 +1,58 @@
+package io.sentry.core;
+
+import static io.sentry.core.ILogger.logIfNotNull;
+
+import java.io.File;
+import org.jetbrains.annotations.Nullable;
+
+public abstract class DirectoryProcessor {
+
+  private ILogger logger;
+
+  protected DirectoryProcessor(@Nullable ILogger logger) {
```

Comment:
we have `NoOpLogger` so it won't be null right

Reply:
fixed: `NotNull`